src/components/RichTextEditor/TipTapEditor.tsx

```diff
@@ -0,0 +1,114 @@
+'use client'
+
+import { useEditor, EditorContent } from '@tiptap/react'
+import StarterKit from '@tiptap/starter-kit'
+import MenuSimple from './components/MenuSimple/MenuSimple'
+import { jsPDF } from 'jspdf'
+import html2canvas from 'html2canvas'
+import './TipTapEditor.css'
+import { cn } from '@/utils/style/cn'
+import { resumeHTML } from '@/data/samples/resume'
+
+const editorContentActual = 'editor-content-actual'
+
+interface TipTapEditorProps {}
+
+const TipTapEditor = (props: TipTapEditorProps) => {
+    const editor = useEditor({
+        extensions: [StarterKit],
+        editorProps: {
+            attributes: {
+                id: editorContentActual,
+                // TODO: We may need to use real world units?  Play around and test
+                // width: 210mm;
+                // height: 297mm;
+                // padding: 20mm;
+                class: cn(
+                    'm-2 bg-white text-black',
+                    'aspect-a4-document', // Aspect ratio of A4 paper
+                    'w-[794px]', // Width of A4 paper in pixels
+                    'm-auto p-[20px]', // Centered and padding inside the editor to simulate margins
+                    // Allows scrolling within the editor if the content exceeds A4 height
+                    // TODO: display a line in the editor representing the end of the page
+                    'overflow-y-auto',
+                    'shadow-md'
+                ),
+            },
+        },
+        content: resumeHTML,
```

Comment:
Could we just pass this dummy HTML in via props from the testEditor?

Reply:
Let's just add `content` to the `TipTapEditorProps`.

Reply:
Yes, totally.  Dependency injection ftw!  I’ll do this in a few.  The editor still needs a lot of work and design feel free to muck with it